### ISSUE TITLE
Some questions about SwinTrans edition

### ISSUE THREAD

**Issue by zhanggtong** [JieShibo/PETL-ViT#4]
I try to use the SwinTrans convpass of the MLP version, but the dimension is always wrong. If dim and Vit_Dim parameter exchange will result in a very large number of parameters, which is far from consistent with the paper. I hope to get an answer. Thank you very much

**Comment by JieShibo** [comment]
Hi, I just checked and ran the code, and there seems to be no problem.
Would you mind giving us more information about the error?
To use the Convpass for Swin, just need to replace https://github.com/JieShibo/PETL-ViT/blob/3b25e37793a2ab97cbf7d06494edb2e222e561d0/vtab/train.py#L66 with 
```python
    state_dict = torch.load('../swin_base_patch4_window7_224_22k.pth')
    model = create_model('swin_base_patch4_window7_224_in22k', drop_path_rate=0.1)
    model.load_state_dict(state_dict['model'])
```
